integration/oasis/stationfinder/stations_iterator_alg.go

```diff
@@ -48,7 +50,7 @@ func CalcCostBetweenChargeStationsPair(from nearbyStationsIterator, to nearbySta
 		startIDs = append(startIDs, v.ID)
 	}
 	if len(startPoints) == 0 {
-		err := fmt.Errorf("Empty iterator of from pass into calcCostBetweenChargeStationsPair")
+		err := fmt.Errorf("Empty iterator of from pass into CalcWeightBetweenChargeStationsPair")
```

Comment:
DON'T use capital letter at the beginning of error string. see https://github.com/golang/go/wiki/CodeReviewComments#error-strings